### DISCUSSION TITLE
release RC 3

### DISCUSSION THREAD

**Discussion by MartijnvanGeffen13** [MartijnvanGeffen13/Volvo4evcc#17] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/MartijnvanGeffen13/Volvo4evcc/releases/tag/release'>release RC 3</a>.</em>

**Comment by MartijnvanGeffen13** [comment]
@erniemctoeff
@Kentrp
@mkarnek
@triple-it

A first release that can be tested is ready, see documentation for instructions.

This is running in my prod for 3 days now stable without issues. Let me know if you run into issues

**Reply by mkarnek** [reply]
Dear @MartijnvanGeffen13, thank you for this build. I put it in production this afternoon and it seems to be working fine. 👍
